src/participants/create_project.py

```diff
@@ -104,7 +104,11 @@ def get_repolinks(self, wid, project):
                 repolinks[repo].append(arch)
             if not repolinks[repo]:
                 del repolinks[repo]
-        return repolinks
+        if repolinks:
+            return repolinks
+        else:
+            # Creating a project with no repos makes no sense
+            raise RuntimeError("No suitable repos found in %s (must contain an arch in the name)" % project)
```

Comment:
To be more specific, create_project is _not just for gating or trials_, but also for normal projects.  These days we don't see that much because our obs is fossilized, but it's still true.